ext/crates/algebra/src/algebra/adem_algebra.rs

```diff
@@ -470,7 +470,7 @@ impl GeneratedAlgebra for AdemAlgebra {
 impl AdemAlgebra {
     pub fn new(p: ValidPrime, generic: bool, unstable: bool, unstable_enabled: bool) -> Self {
         assert!(unstable_enabled || !unstable);
-        fp::vector::initialize_limb_bit_index_table(p);
+        // fp::vector::initialize_limb_bit_index_table(p);
```

Comment:
It's better to remove code rather than comment it out, here and in other places.
```suggestion
```